cmd/fs-v1-metadata.go

```diff
@@ -249,8 +249,7 @@ func (m *fsMetaV1) ReadFrom(ctx context.Context, lk *lock.LockedFile) (n int64,
 	}
 
 	if len(fsMetaBuf) == 0 {
-		logger.LogIf(ctx, io.EOF)
-		return 0, io.EOF
```

Comment:
Returning io.EOF is not a problem, returning error is compatible with io.ReaderFrom interface requirements. We simply have to ignore io.EOF at the caller level and proceed forward.